### PR TITLE
Adjusted the readme to show the permission required clearly

### PR DESCRIPTION
**Pull request by zulandar**
Currently the permissions aren't showing clearly within the readme file along with the permission change from v1 to v2 customer portal.

### CODE REVIEW

**Comment by SonarSoftware** [comment]
Rather than making this change, why don't we capture the permission failure if it happens and display a human readable error on the settings page?

**Comment by zulandar** [comment]
We could do that. I can submit a PR for that but I still feel the readme needs to be in place to assist with initial setup steps. In the initial readme it feels almost hidden.

**Comment by zulandar** [comment]
#24 is the issue to track the /settings page error when API user permissions is incorrect.

**Comment by zulandar** [comment]
Reference #27 PR for the error page.

File: README.md [hunk 1]
```diff
@@ -39,7 +39,13 @@ You can view the installation log by running
 
 ### A note on initial setup
 
-The API username and API password you are prompted for are credentials for your Sonar instance. You should create a dedicated user to utilize for the customer portal - **do not use your admin username/password!** The user requires account create, read, update, and delete permissions, ticket create, read, and update permissions, ticket super user permissions, and financial read permissions.
+The API username and API password you are prompted for are credentials for your Sonar instance. You should create a dedicated user to utilize for the customer portal - **do not use your admin username/password!** 
+
+**This permissions are different then v1 portal permissions**
```
Comment (SonarSoftware) [inline README.md]: `This` should be `These`. `Then` should be `than`.

Comment (zulandar) [reply README.md]: Corrected

**Pull request merged**